infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/runtime/LogReplicationClientRouter.java

```diff
@@ -179,6 +180,7 @@ public IClientRouter addClient(IClient client) {
             @Nonnull String nodeId) {
 
         HeaderMsg.Builder header = HeaderMsg.newBuilder()
+                .setVersion(getDefaultProtocolVersionMsg())
```

Comment:
Is there anything server-side that needs to be adjusted similarly?

Reply:
No, I couldn't find it in any other place. In the server, we merge from the client's header and set the server version explicitly.